filter/ArticleMedraXmlFilter.inc.php

```diff
@@ -269,7 +269,13 @@ function createContentItemNode($doc, $issue, $article, $galley, $objectLocalePre
 			}
 		}
 		// Article Title (mandatory)
-		$titles = $this->getTranslationsByPrecedence($article->getCurrentPublication()->getFullTitles(), $objectLocalePrecedence);
+		$titles = $this->getTranslationsByPrecedence(
+			array_map(
+				fn($pubTitle) => htmlspecialchars($pubTitle, ENT_COMPAT, 'UTF-8'), 
+				$article->getCurrentPublication()->getFullTitles('html')
```

Comment:
Does MEDRA support HTML? I don't see it documented in pkp/pkp-lib#2564 or at a glance in the schema (http://www.medra.org/schema/onix/DOIMetadata/2.0/ONIX_DOIMetadata_2.0.xsd).

Reply:
Maybe just to double check with you: if the value is xs:string, does it mean that html elements are allowed? -- Validator does not complain, but...

Reply:
Hmmm... I saw another comment... So we do not want to provide `E=mc&lt;sup&gt;2&lt;/sup&gt;` only if the schema really defines what elements...
In that case, Touhidur and Alec, you were right: DataCite, mEDRA and DOAJ would not support it.
But I think PubMed does (http://www.ncbi.nlm.nih.gov/entrez/query/static/PubMed.dtd) ?

Reply:
PubMed's title element (from the DTD you linked above) is defined as...
```
<!ELEMENT ArticleTitle     (%data; | mml:math)*>
```
...so I don't see italics/bold/underline, but it does support `<mml:math>` elements (see https://jats.nlm.nih.gov/archiving/tag-library/1.2/element/mml-math.html). While we could maybe find a way to map everything we currently support into mml:math, I'd argue that's probably not a good approach. I'd recommend we go with plaintext for PubMed.

Reply:
But isn't `%data`: `<!ENTITY % data             "#PCDATA | sup | inf | b | u | i | sub">`

Reply:
Ah! Yes, you're absolutely right. @touhidurabir, PubMed *does* support `sup`/`inf`/`b`/`u`/`i`/`sub` after all.